wire/addr.go

```diff
@@ -210,3 +213,42 @@ func (addr *Address) Equal(other *Address) bool {
 		addr.Nonce == other.Nonce &&
 		addr.Signature.Equal(&other.Signature)
 }
+
+// DecodeString into a wire-compatible Address.
+func DecodeString(addr string) (Address, error) {
+	// Remove any leading slashes.
+	if addr[:1] == "/" {
+		addr = addr[1:]
+	}
```

Comment:
```suggestion
	// Remove any leading slashes.
	if strings.HasPrefix(addr, "/") {
		addr = addr[1:]
	}
```
Otherwise, we have to manually check for edge cases like the empty string (which was not previously being checked).